tasks-by-users/AbbasShaikh0304/p1/index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+  <html>
+      <head>
+          <meta charset="utf-8">
+          <meta name="keywords" content="Automobile, parts, garage">
+          <meta name="Description" content="Welcome to our garage to get parts at most affordable prizes">
+          <title>Welcome to Online Book Store</title>
+      </head>
+      <body>
+          <header><img src="/img/Logo.png" width="100%" height="150px" alt="oops!"></header>
+          <h1>
+            Online Book Store     
+          </h1>
+          <p>Welcome to <strong>online Book Store</strong>.We sell the Books of varies genre at most affordable prize. You can also download the digital copy of your Books.</p>
+          <h2>Genre</h2>
+          <hr>
+              <ul>
+                          <li><img src="/img/Comic.jpg" width="100px" height="100px"><a href="">Comic</a></li>
+                          <li><img src="/img/Biography.jpg" width="100px" height="100px"><a href="">Biography</a></li>
+                          <li><img src="/img/Fiction.jpg" width="100px" height="100px"><a href="">Fiction</a></li>
+                          <li><img src="/img/Historical.jpg" width="100px" height="100px"><a href="">Historical</a></li>
+              </ul>
+              <hr>
+              <h3>Join our prime membership and get the latest updates first!</h3>
+              <button>Join Now!</button>
+              <ul>
+                  <li>
+                  Follow Us
+                <ul>
+                    <li><a href="">FaceBook</a></li>
+                    <li><a href="">Twitter</a></li>
+                    <li><a href="">YouTube</a></li>
+                    <li><a href="">Instagram</a></li>
+                </ul>
```

Comment:
why `ul` in `li`

Reply:
It is list inside an another list. Check full code